state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -898,6 +903,22 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Update the fee multiplier with the provided value
+		///
+		/// Requires Governance.
+		#[pallet::call_index(7)]
+		#[pallet::weight(T::WeightInfo::ccm_broadcast_failed())]
+		pub fn update_fee_multiplier(
```

Comment:
We can add this to the PalletConfigUpdate enum and handle it there instead of defining a new extrinsic.